### PR TITLE
routerLinks

### PR DESCRIPTION
**Pull request by elmerv**
This PR currently adds routerLinks on SuperBar.

### CODE REVIEW

**Comment by elmerv** [comment]
> Note:
> I spammed clicks on each tab click you see in the demos.

https://github.com/solid-adventure/trivial-ui/assets/20259244/7261adef-a9fb-4133-8098-a2a71be3e32b

https://github.com/solid-adventure/trivial-ui/assets/20259244/da88cfbf-2bc9-4652-8278-a9a494c16499

File: src/components/Main.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
     <!-- Main content area where router views are rendered -->
     <main>
-      <router-view class="clearSuperbar"></router-view>
+      <router-view :key="$route.fullPath" class="clearSuperbar"></router-view>
```
Comment (elmerv) [inline src/components/Main.vue:11]: This ensures that different route same component works. Ex. AppOverview

Comment (elmerv) [reply src/components/Main.vue:11]: Link to resource I got this from: https://stackoverflow.com/questions/42603583/vue-js-same-component-with-different-routes

**Review by vgkids — COMMENTED** [review]
@elmerv Looks great, please proceed to replace the rest of the `<a href` links.

File: src/router/index.js [hunk 2]
```diff
@@ -101,6 +106,14 @@ const routes = [
     component: OrganizationInviteUser,
     name: "Invite User",
   },
+  {
+    path: "/organizations/:pathMatch(.*)*",
```
Comment (elmerv) [inline src/router/index.js:110]: This prevents router warn logs when running resolve on each subpath. This is done in main.

File: src/router/index.js [hunk 3]
```diff
@@ -91,6 +91,11 @@ const routes = [
   { path: "/apps/:id/builder2/*",
     component: Builder,
     name: "Actions" }, //change this path
+  {
+    path: "/organizations",
```
Comment (elmerv) [inline src/router/index.js:95]: adding path on line 110 resulted in breadcrumb of PanelType, this introduces a more appropriately named breadcrumb of Organizations which redirects to settings.
Screen Shot 2024-03-20 at 11 29 28 AM
Screen Shot 2024-03-20 at 11 29 49 AM

File: src/components/Main.vue [hunk 4]
```diff
@@ -95,13 +99,12 @@ export default {
...
         allBreaks.push(url);
       }
-
       let resolvedRoutes = allBreaks.map((x) =>
         this.$router.resolve({ path: x })
       );
 
       this.breadcrumbs = resolvedRoutes
-        .filter((x) => x.matched && (x.matched.length ?? 0) > 0)
+        .filter((x) => (x.matched && (x.matched.length ?? 0) > 0) && x.name)
```
Comment (elmerv) [inline src/components/Main.vue:107]: Does not add paths with undefined names in breadcrumbs.

Comment (elmerv) [reply src/components/Main.vue:107]: Prevents this from happening:
Screen Shot 2024-03-20 at 11 44 25 AM

**Review by vgkids — APPROVED** [review]
Looks great.

I pushed a change to SuperBar isActive; your version may have worked, I mistakenly tested yours without having all of your commits and made my own version.

Given the two, let's go for the simple one-liner.

**Comment by vgkids** [comment]
@elmerv Since Vue3 we had an issue where you would drop changes if you visited different actions between hitting Save. This addresses that by having the App Builder in the nav tree reset the outermost action without reloading the page. This also adds a simple breadcrumb to perform the same thing.

Screenshot 2024-03-22 at 1 01 16 PM

**Pull request merged**